cetmix_tower_webhook/readme/CONFIGURE.md

```
@@ -0,0 +1,52 @@
+- Enabled. Uncheck this field to disable the webhook without deleting it
+- Name. Authenticator name
+- Reference. Unique reference. Leave this field blank to auto generate it
+- Authenticator. Select an Authenticator used for this web hook
+- Endpoint. Webhook andpoint. The complete webhook URL will be <your_tower_url>/cetmix_tower_webhooks/<endpoint>​
+- Run as User. Select a user to run the webhook from behalf of. CAREFUL! You must realise and understand what you are doing including all the possible consequences when selecting a specific user.
+- Code. Code that is used to authenticate the request. You can use all Cetmix Tower - Python command variables except for the server​ plus the following webhook specific ones:
+  - payload: dictionary that contains the json payload or the GET parameters of the request
+
+Webhook code returns a result using the Cetmix Tower Python command pattern:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)